README.md

```diff
@@ -1,12 +1,25 @@
-[![Apache License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-# YouTube Analytics
-
-This package models YouTube Analytics data from [Fivetran's connector](https://fivetran.com/docs/applications/youtube-analytics#youtubeanalytics). It uses data in the format described by the [YouTube Channel Report schemas](https://fivetran.com/docs/applications/youtube-analytics#schemainformation).
-
-The main focus of the package is to transform the core object tables into analytics-ready models. It includes options to explore video demographics and a comprehensive overview of video performance that you could combine with other organic ad platform reports.
-
-## Models
-This package contains transformation models, designed to work simultaneously with our [YouTube Analytics source package](https://github.com/fivetran/dbt_youtube_analytics_source). A dependency on the source package is declared in this package's `packages.yml` file, so it will automatically download when you run `dbt deps`. The primary outputs of this package are described below.
+<p align="center">
+    <a alt="License"
+        href="https://github.com/fivetran/dbt_youtube/blob/main/LICENSE">
+        <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" /></a>
+    <a alt="Fivetran-Release"
+        href="https://fivetran.com/docs/getting-started/core-concepts#releasephases">
+        <img src="https://img.shields.io/badge/Fivetran Release Phase-_Beta-orange.svg" /></a>
+    <a alt="dbt-core">
+        <img src="https://img.shields.io/badge/dbt_Core™_version->=1.0.0_,<2.0.0-orange.svg" /></a>
+    <a alt="Maintained?">
+        <img src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" /></a>
+    <a alt="PRs">
+        <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
+</p>
+# YouTube Analytics Transformation dbt Package ([Docs](https://fivetran.github.io/dbt_youtube_analytics/))
+# 📣 What does this dbt package do?
+- Produces modeled tables that leverage YouTube Analytics data from [Fivetran's connector](https://fivetran.com/docs/applications/youtube-analytics#youtubeanalytics) in the format described by [this ERD](https://fivetran.com/docs/applications/youtube#schemainformation) and build off the output of our [YouTube Analytics source package](https://github.com/fivetran/dbt_youtube_analytics_source).
+
+It includes options to explore video demographics and a comprehensive overview of video performance that you could combine with other organic ad platform reports.
+
+The following table provides a detailed list of all models materialized within this package by default. 
+> TIP: See more details about these models in the package's [dbt docs site](https://fivetran.github.io/dbt_youtube/#!/overview?g_v=1&g_e=seeds).
```

Comment:
For the table below can you add the dbt docs links instead of the model files.
```md
| **model**                 | **description**                                                                                                    |
| ------------------------- | ------------------------------------------------------------------------------------------------------------------ |
| [youtube__video_report](https://fivetran.github.io/dbt_youtube_analytics/#!/model/model.youtube_analytics.youtube__video_report)  | Each record represents the daily aggregation of your YouTube video performance.    |
| [youtube__demographics_report](https://fivetran.github.io/dbt_youtube_analytics/#!/model/model.youtube_analytics.youtube__demographics_report)        | Each record represents a daily video view percentage by gender, age, and country.            |
| [youtube__age_demographics_pivot](https://fivetran.github.io/dbt_youtube_analytics/#!/model/model.youtube_analytics.youtube__age_demographics_pivot)        | Each record represents a daily video view percentage with the age ranges pivoted out for quicker analysis.            |
| [youtube__gender_demographics_pivot](https://fivetran.github.io/dbt_youtube_analytics/#!/model/model.youtube_analytics.youtube__gender_demographics_pivot)        | Each record represents a daily video view percentage with the gender options pivoted out for quicker analysis.            |
| [youtube__video_metadata](https://fivetran.github.io/dbt_youtube_analytics/#!/model/model.youtube_analytics.youtube__video_metadata)           | Each record represents an individual video enriched with metadata. This model is disabled by default. Read our [YouTube Video Metadata Cloud Function Write Up](https://resources.fivetran.com/datasheets/youtube-metadata-cloud-function-guide-2) or our [YouTube Analytics source package README](https://github.com/fivetran/dbt_youtube_analytics_source) to learn how to generate this model.          |
```